File: src/main/java/me/creepinson/creepinoutils/api/util/data/game/IWorld.java
package me.creepinson.creepinoutils.api.util.data.game;

import me.creepinson.creepinoutils.api.util.ISerializable;

import java.util.Collection;

/**
 * @author Creepinson https:/theoparis.com/about
 **/
public interface IWorld extends ISerializable {
    Collection<IChunk> getLoadedChunks();

    IChunk getChunk(int x, int z);

    int getDimension();
}

File: src/main/java/dev/throwouterror/util/data/game/IWorld.kt
package dev.throwouterror.util.data.game

import dev.throwouterror.util.ISerializable

/**
 * @author Creepinson https:/theoparis.com/about
 */
interface IWorld : ISerializable<Any?> {
    val loadedChunks: Collection<IChunk?>?
    fun getChunk(x: Int, z: Int): IChunk?
    val dimension: Int
}